### DISCUSSION TITLE
Connection to Oracle Database

### DISCUSSION THREAD

**Discussion by TomaszMyszak** [dotnet/maui#15543] [General]
Hi, 
how to get connection on Oracle DataBase in MAUI project. I try install form NuGet
`NuGet\Install-Package Oracle.ManagedDataAccess -Version 21.10.0`

I get issue:

+ NuGet\Install-Package Oracle.ManagedDataAccess -Version 21.10.0
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand

I try using Oracle.ManagedDataAccess.Client;

Can anyone help me?

**Comment by TomaszMyszak** [comment]
I get it, oracle don't support .Net 7.0

**Reply by Xyncgas** [reply]
Have you tried writing code in C++ to do whatever you want and use C++ code in .NET via native interop
